src/scaffold/vap.rs

```diff
@@ -115,7 +130,7 @@ fn convert_vap_to_cluster_admission_policy(
             mutating: false,
             background_audit: true,
             context_aware_resources: BTreeSet::new(),
-            failure_policy: vap_spec.failure_policy,
+            failure_policy: None,
```

Comment:
pseudo random location, it would be nice to have a test that covers the failurePolicy conversion